server/models/remote_provider.go

```diff
@@ -111,7 +111,10 @@ func (l *RemoteProvider) loadCapabilities(token string) {
 		// Proceed to make a request with the token
 		resp, err = l.DoRequest(req, token)
 	}
-
+	if resp == nil {
+		logrus.Errorf(ErrUnreachableRemoteProvider(err))
```

Comment:
```suggestion
		logrus.Errorf(ErrUnreachableRemoteProvider(err).Error())
```

Build is failing, please ensure build completes at your local machine before pushing changes.